module/VuFind/src/VuFind/View/Helper/Root/SchemaOrg.php

```diff
@@ -106,4 +109,28 @@ public function getMeta(string $property, string $content, array $attributes = [
     {
         return $this->getTag('meta', compact('property', 'content') + $attributes);
     }
+
+    /**
+     * Get all record types for the given record.
+     *
+     * @param DefaultRecord $driver Record Driver
+     *
+     * @return array
+     */
+    public function getRecordTypesArray(DefaultRecord $driver): array
```

Comment:
It might be more open-ended to use \VuFind\RecordDriver\AbstractBase as the type instead of DefaultRecord, and then change the code to: `return $driver->tryMethod('getSchemaOrgFormatsArray') ?? [];` to be sure that nothing strange happens if an exotic record driver gets passed to the helper. (I think that's unlikely to happen, but it doesn't hurt to prepare for the possibility).

Reply:
I had a few minutes this afternoon, so I went ahead and updated the unit test for the SchemaOrg helper. I realized that doing this required my above suggested changes since the test harness for record drivers is not a DefaultRecord subclass, so I went ahead and applied those changes. Please take a look and let me know what you think!

Reply:
Thanks, looks good to me!